vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/RadioButtonSteps.java

```diff
@@ -86,6 +95,38 @@ public WebElement assertIfRadioOptionExists(String radioOption)
                 new Locator(WebLocatorType.XPATH, "input[@type='radio']"));
     }
 
+    /**
+     * Checks that a <b>radio button</b> specified by the <b>name</b> exists within
+     * the previously specified <b>search context</b>
+     * <p>A <b>radio button</b> is an <i><code>&lt;input&gt;</code></i> element with an attribute
+     * 'type' = 'radio' and a <b>name</b> for it is a 'text' or any 'attribute value' of
+     * it's <i><code>&lt;label&gt;</code></i> element (<i><code>&lt;label&gt;</code></i>
+     * with an attribute 'for' = radio button id).
+     * </p>
+     * <p>Actions performed at this step:</p>
+     * <ul>
+     * <li>Finds the <b>radio button</b> specified by the <b>name</b> in the radio group;</li>
+     * </ul>
+     * @param radioOption A name of the <b>radio button</b>
+     * @return WebElement An <b>element</b> matching the requirements,
+     * <b> null</b> - if there are no desired elements.
+     * <b>Example:</b>
+     * <pre>
+     * {@code <div>
+     *      <label>}<b>'radioGroup'</b>{@code </label>
+     *      <div>
+     *          <input id="radioButtonId" type="radio" />
+     *          <label for="radioButtonId">}<b>'radioOption'</b>{@code </label>
+     *      </div>
+     * </div>}</pre>
+     */
+    @Then("radio button with name `$radioOption` exists")
+    public WebElement assertIfRadioOptionExists(String radioOption)
+    {
+        return baseValidations.assertElementExists(String.format("A radio button with name '%s'", radioOption),
+                new Locator(FIELD_NAME, radioOption)).orElse(null);
```

Comment:
it's not correct use only fieldName locator, it's doesn't check for `type` of `input`